src/Model/CampaignDraft.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Mailjet\MailjetBundle\Model;
+
+/**
+ * https://dev.mailjet.com/email-api/v3/campaigndraft/
+ */
+class CampaignDraft {
+
+    const LOCALE_KEY = 'Locale';
+    const SENDER_KEY = 'Sender';
+    const SENDEREMAIL_KEY = 'SenderEmail';
+    const SUBJECT_KEY = 'Subject';
+    const CONTACTLISTID_KEY = 'ContactsListID';
+
+    //Mandatory properties (must be set through constructor)
+    protected $locale;
+    protected $sender;
+    protected $senderEmail;
+    protected $subject;
+    protected $contactsListID;
+    //Optional properties (array)
+    protected $optionalProperties = null;
+    //content
+    protected $content = null;
+    //id
+    protected $id = null;
+
+    /**
+     * @param $Locale
+     * @param $Sender
+     * @param $SenderEmail
+     * @param $Subject
+     * @param $ContactsListID
+     * @param optionalProperties
+     */
+    public function __construct($locale, $sender, $senderEmail, $subject, $contactsListID, $optionalProperties = null) {
```

Comment:
Humm, here we need a model layer in order to abstract the API. If we add new feature such as campaign in viewTable for example, we will use CampaignDraft Model. The Model should manage the compatibility with the API. For example, I'm using CampaignDraft Model in my app. If the API change, we have to update the CampaignDraft Model so users only have to composer update our dependencies and modify their code everytime.